src/renderer/components/open-project-page/open-project-page.tsx

```diff
@@ -38,6 +52,21 @@ export const OpenProjectPage: FC<{ onNext: Consumer<void>; onBack: MouseEventHan
     }, [error]);
   }
 
+  const useResumeCheck = () => {
```

Comment:
Because all the changes are in the renderer process, make it a `util` or `service` and implement the logic there.